torchaudio/models/_wavernn.py

```diff
@@ -50,8 +50,8 @@ class _MelResNet(nn.Module):
     Args:
         n_res_block: the number of ResBlock in stack (default=10)
         n_freq: the number of bins in a spectrogram (default=128)
-        n_hidden: the number of hidden dimensions (default=128)
-        n_output: the number of output dimensions (default=128)
+        n_hidden_resblock: the number of hidden dimensions of resblock (default=128)
+        n_output_melresnet: the number of output dimensions of melresnet (default=128)
```

Comment:
These are clear as they are for a MelResBlock. Let's keep `n_hidden` and `n_output` here, but add "of resblock" and "of melresnet" in the description as you have done here.

Reply:
Yes, I keep them as before.